system-int-test/src/cmr/system_int_test/utils/logging_util.clj

```diff
@@ -0,0 +1,52 @@
+(ns cmr.system-int-test.utils.logging-util
+  "This contains utilities for testing logging"
+  (:require
+   [clojure.test :refer [is]]
+   [cmr.common-app.api.routes :as cr]
+   [cmr.common.mime-types :as mt]
+   [cmr.mock-echo.client.echo-util :as e]
+   [cmr.system-int-test.system :as s]
+   [cmr.transmit.logging :as trans-logging]))
```

Comment:
With this setup we aren't doing true system integration tests. The way the other tests work we hit a URL defined in a url-helper namespace. That way if a route changes in transmit our tests will fail (we broke backwards compatibility). The way this is written we're completely ignoring the API call and just using the source code to make the right call.